src/main/resources/init_db.sql

```diff
@@ -0,0 +1,9 @@
+CREATE DATABASE `taxi_db`;
```

Comment:
Is there a reason to call this database `taxi_db`? Are you planning on adding other tables? If not, I suggest renaming it to `manufacturers`.